lib/rubocop/config_obsoletion.rb

```diff
@@ -19,6 +19,7 @@ class ConfigObsoletion
       'Layout/LeadingBlankLines' => 'Layout/LeadingEmptyLines',
       'Layout/TrailingBlankLines' => 'Layout/TrailingEmptyLines',
       'Lint/DuplicatedKey' => 'Lint/DuplicateHashKey',
+      'Lint/EndInMethod' => 'Style/EndBlock',
```

Comment:
Can you arrange them in ASCII order?

```suggestion
      'Lint/EndInMethod' => 'Style/EndBlock',
      'Lint/DuplicatedKey' => 'Lint/DuplicateHashKey',
```

Reply:
Oh, this is my mistake 💦